scripts/superdesk/upload/styles/upload.less

```diff
@@ -485,13 +486,24 @@ media uploader
 
 					input[type="text"] {
 						border: 0 !important;
-						height: 34px !important;
+						height: 26px !important;
```

Comment:
Using `!important` is often a bad idea. It's better to play with selectors precision.
In my side, I remove them every time I see one :)